media-gfx/digikam/digikam-5.3.0.ebuild

```diff
@@ -0,0 +1,166 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+if [[ ${KDE_BUILD_TYPE} != live ]]; then
+	KDE_HANDBOOK="true"
+	KDE_TEST="true"
+fi
+CMAKE_MAKEFILE_GENERATOR="emake"
+CMAKE_MIN_VERSION="3.0"
+inherit kde5 toolchain-funcs
+
+DESCRIPTION="Digital photo management application"
+HOMEPAGE="https://www.digikam.org/"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="addressbook calendar gphoto2 jpeg2k +kipi +lensfun marble semantic-desktop mysql opengl openmp +panorama scanner video X"
+
+if [[ ${KDE_BUILD_TYPE} != live ]]; then
+	MY_PV=${PV/_/-}
+	MY_P=${PN}-${MY_PV}
+	SRC_BRANCH=stable
+	[[ ${PV} =~ beta[0-9]$ ]] && SRC_BRANCH=unstable
+	SRC_URI="mirror://kde/${SRC_BRANCH}/digikam/${MY_P}.tar.xz"
+	S="${WORKDIR}/${MY_P}/core"
+fi
+
+COMMON_DEPEND="
+	$(add_frameworks_dep kconfig)
+	$(add_frameworks_dep kconfigwidgets)
+	$(add_frameworks_dep kcoreaddons)
+	$(add_frameworks_dep kiconthemes)
+	$(add_frameworks_dep ki18n)
+	$(add_frameworks_dep kio)
+	$(add_frameworks_dep knotifications)
+	$(add_frameworks_dep knotifyconfig)
+	$(add_frameworks_dep kservice)
+	$(add_frameworks_dep kwidgetsaddons)
+	$(add_frameworks_dep kwindowsystem)
+	$(add_frameworks_dep kxmlgui)
+	$(add_frameworks_dep solid)
+	$(add_qt_dep qtdbus)
+	$(add_qt_dep qtgui '-gles2')
+	$(add_qt_dep qtprintsupport)
+	$(add_qt_dep qtsql 'mysql?')
+	$(add_qt_dep qtwebkit)
+	$(add_qt_dep qtwidgets)
+	$(add_qt_dep qtxml)
+	dev-libs/expat
+	media-gfx/exiv2:=
+	media-libs/lcms:2
+	media-libs/liblqr
+	>=media-libs/libpgf-6.12.27
+	media-libs/libpng:0=
+	media-libs/opencv:=[-qt4]
+	|| ( <media-libs/opencv-3.0.0 >=media-libs/opencv-3.1.0 )
```

Comment:
The order is correct in this case. digikam-5.3.0 will be stabilised long before any version of opencv-3 is going to be.